packages/scandipwa/src/store/Wishlist/Wishlist.dispatcher.ts

```diff
@@ -36,7 +36,7 @@ import { ClearWishlistAction, WishlistProduct } from './Wishlist.type';
 
 /**
  * Get wishlist setting.
- * @namespace Store/Wishlist/Dispatcher/isWishlistEnabled */
+ /* @namespace Store/Wishlist/Dispatcher/isWishlistEnabled */
```

Comment:
No need to make it a separate comment.
Or add a closing of the previous comment.

Reply:
+